networking.go

```diff
@@ -191,3 +195,81 @@ func startSSFUnix(s *Server, addr *net.UnixAddr) <-chan struct{} {
 	}()
 	return done
 }
+
+type SoftnetData struct {
```

Comment:
This makes logical sense here, but perhaps it's better placed in something os metric centric package-wise?